### ISSUE TITLE
NUX page control "dots" aren't selectable

### ISSUE THREAD

**Issue by kwonye** [Automattic/simplenote-ios#14]
The dots at the bottom of the screen in the NUX should be selectable to move between pages.  This is an accessibility issue as well as general adherence to standard control usage.

![ios simulator screen shot dec 8 2014 9 23 26 am](https://cloud.githubusercontent.com/assets/373903/5341468/ede743b2-7ebb-11e4-86b1-ac63ea05dbe0.png)

They seem to be somewhat selected when running on an actual device.  Not sure what's actually going on or if there is something obscuring a part of the `UIPageControl`.

Opened by @astralbodies

**Comment by bummytime** [comment]
We definitely could make this more accessible:

<img width="659" alt="Screen Shot 2019-05-14 at 1 03 30 PM" src="https://user-images.githubusercontent.com/154014/57721090-ee585500-7648-11e9-8b14-759a76115231.png">

**Comment by jleandroperez** [comment]
Closing this one since this UI has been deprecated.
